crown-court-proceeding/src/main/resources/db/changelog/changeset/12-create-dead-letter-message-table.sql

```diff
@@ -0,0 +1,10 @@
+--liquibase formatted sql
+--changeset Karl Baker:12-dead-letter-message-table.sql
+CREATE SEQUENCE  IF NOT EXISTS DEAD_LETTER INCREMENT BY 1 START WITH 1;
+
+CREATE TABLE IF NOT EXISTS crown_court_proceeding.DEAD_LETTER_MESSAGE
+(   ID INTEGER NOT NULL DEFAULT nextval('crown_court_proceeding.DEAD_LETTER') PRIMARY KEY,
+    MESSAGE BYTEA,
```

Comment:
Happy to do either or. Will plan on changing to JSON type, but happy to also add a MAAT ID for easy look-up.

Reply:
Yes, I think this is probably redundant if we're storing the data as JSON because it's easy to directly query the contents of the message and extract the ID.